.travis.yml

```diff
@@ -81,3 +81,4 @@ jobs:
     - stage:
       <<: *tox
       env: TOXENV=dennis-lint
+  after_success: yarn storybook-deploy -- --ci
```

Comment:
Because we use yarn, I don't think we need `--`, we can directly pass arguments to the script:

```
yarn storybook-deploy --ci
```

In addition, we'll need a condition to only run `after_success` from the `master` branch.

Reply:
I added a condition. let me know if this works, otherwise I might need your help in adding a "stage"

Reply:
We should not have such a condition right now, because we are not able to test that way.

I'd suggest you to remove your last commit, close this PR, push your branch directly here and open a PR. It should deploy the storybook to `gh-pages`. If that works, you'll have to add the condition to only deploy on `master`.